j2me/communication/src/org/javarosa/service/transport/securehttp/AuthenticatedHttpTransportMessage.java

```diff
@@ -241,6 +250,30 @@ public void send() {
             }
         }
     }
+    
+    private void attemptBrowserCertificateAcceptCallout() {
+    	try {
+	    	if(CalloutResolver != null) {
+	    		//See if we have a location in our current context to do a cert accept
+	    		String fetchUrl = 
+	    				PropertyManager._().getSingularProperty(TransportPropertyRules.HTTP_CERTIFICATE_REQUEST_URL);
+	    		
+	    		if(fetchUrl == null) {
+	    			fetchUrl = URL;
```

Comment:
URL is an existing class member variable that is the URL for the Transport Message itself